releasenotes/notes/profiling-default-timeout-agent-43b19237a21fd337.yaml

```diff
@@ -2,4 +2,4 @@
 other:
   - |
     The default connection timeout for the profiling agent has now been reduced
-    from 10 to 2 seconds to match the tracer behaviour.
+    from 10 to 2 seconds to match the tracer behavior.
```

Comment:
:cry: 🇨🇦 🇬🇧 🇪🇺

Reply:
Is it possible to support both?

Personally the localization differences don't really make much of a difference to me. Would be better to allow either ?

Reply:
It'd certainly increase the CI pass rate 🙂

Reply:
Je peux écrire en français sinon.